### DISCUSSION TITLE
show next result while searching

### DISCUSSION THREAD

**Discussion by mhash17** [johnfactotum/foliate#895] [Q&A]
Hi, when i look for a term (ctl+f), how can i jump to the next result without clicking on the search icon and then click the next result and so on. With other applications when searching you just push the tab-key to get the next result, but foliate does not stay in the search dialog. so for every next result you have to click click click, that's cumbersome. Isn't there a way to just search (ctl+f, tab (next result), shift+tab (prev. result)) or some other form like ctl+k / ctl+n?

**Comment by johnfactotum** [comment]
Currently, there isn't a way. See #693 and #674.
